pkg/controller/kubedirectorcluster/types.go

```diff
@@ -95,6 +95,11 @@ const (
 	2> /opt/guestconfig/configure.stderr  &`
 	fileInjectionCommand = `mkdir -p %s && cd %s &&
 	curl -L %s -o %s`
+	appPrepConfigRunCmdTest = `rm -f /opt/guestconfig/configure.* &&
```

Comment:
will we lose previous guest config logs by removing this?

Reply:
currently, yes